pandas/plotting/_core.py

```diff
@@ -1468,9 +1468,13 @@ def scatter(self, x, y, s=None, c=None, **kwargs):
         y : int or str
             The column name or column position to be used as vertical
             coordinates for each point.
-        s : scalar or array_like, optional
+        s : str, scalar or array_like, optional
             The size of each point. Possible values are:
 
+            .. versionchanged:: 1.1.0
```

Comment:
Typically these `versionchanged` are put at the end up the description for this parameter. SO around line 1483